services/apps/alcs/src/portal/application-submission-review/application-submission-review.controller.ts

```diff
@@ -210,7 +214,7 @@ export class ApplicationSubmissionReviewController {
   ) {
     if (primaryContact.email) {
       const status = await this.applicationSubmissionService.getStatus(
-        APPLICATION_STATUS.IN_REVIEW,
+        SUBMISSION_STATUS.IN_REVIEW_BY_FG, // TODO is this a correct assumption? previously we only had one in review
```

Comment:
Have we figured this out? I only see a single in review status.

Reply:
yes, it correct one, forgot to delete TODO